components/log-viewer-webui/client/src/pages/IngestPage/sqlConfig.ts

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+
+
+/**
+ * Sql table names.
+ */
+const SQL_CONFIG = Object.freeze({
+    SqlDbClpArchivesTableName: "clp_archives",
+    SqlDbClpFilesTableName: "clp_files",
+    SqlDbCompressionJobsTableName: "compression_jobs",
+});
```

Comment:
can we use an enum for this
```suggestion
enum SQL_CONFIG {
    SqlDbClpArchivesTableName = "clp_archives",
    SqlDbClpFilesTableName = "clp_files",
    SqlDbCompressionJobsTableName = "compression_jobs",
}
```